language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/command/executors/CreateFunctionExecutor.java

```diff
@@ -198,6 +194,24 @@ public Object execute(LSContext context) throws LSCommandExecutorException {
         return applyWorkspaceEdit(Collections.singletonList(Either.forLeft(textDocumentEdit)), client);
     }
 
+    private BLangCompilationUnit getCurrentCUnit(LSContext context, BLangPackage packageNode) {
+    /*
+    In windows platform, relative file path key components are separated with "\" while antlr always uses "/"
```

Comment:
With the removal of ANTLR, may be we can optimize this part

Reply:
Yes, we might be able to optimize this. Shall we add this in a separate PR since we have the same logic in many places?

Reply:
+1 let's do this after an API revamp